t/ui/18-tests-details.t

```diff
@@ -563,6 +563,48 @@ subtest ansiToHtml => sub {
 
 my $t = Test::Mojo->new('OpenQA::WebAPI');
 
+subtest 'GRU dependency shown in infopanel' => sub {
+    my $job = $jobs->find(99927);
+    $job->update({state => SCHEDULED, result => NONE});
+
+    my $task = $schema->resultset('GruTasks')->create(
+        {
+            taskname => 'git_clone',
+            args => {},
+            run_at => DateTime->now(),
+            priority => 0,
+        });
+
+    my $gru_dep = $schema->resultset('GruDependencies')->create(
+        {
+            job_id => $job->id,
+            gru_task_id => $task->id,
+        });
+
+    $driver->get('/tests/' . $job->id);
+    disable_bootstrap_animations;
+    wait_for_ajax msg => 'job with GRU dependency loaded';
+
+    my $state_regex = qr{State: <b>.*</b>, waiting for background tasks \(<a href=.*note=gru_id_1};
```

Comment:
I would avoid matching literal HTML if not necessary. If a small thing changes, you have to adapt all the tests.
Also, so far I can't find a usage of `innerHTML` in any of our tests.

And I would actually use the `$task->id` instead of the hardcoded 1, it was just a quick code suggestion where I used it.

Reply:
> innerHTML is a property of an element in the dom, should I use stuff only that has already been used?

No, but that's not the point.
My point is, there is maybe a reason, why we don't use innerHTML in tests yet, and I stated my reason:
> I would avoid matching literal HTML if not necessary. If a small thing changes, you have to adapt all the tests.

Reply:
And @perlpunk's suggestion is also in-line with my review comment where I also suggested alternatives.